docs/sources/setup/install/helm/install-monolithic/_index.md

```diff
@@ -10,107 +10,411 @@ weight: 100
+## Prerequisites
+
+- Helm 3 or above. See [Installing Helm](https://helm.sh/docs/intro/install/).
+- A running Kubernetes cluster.
+
+## Single Replica or Multiple Replicas
+
+There are two ways to deploy Loki in monolithic mode:
+1. **Single Replica**: Run Loki with a single replica. This mode is useful for testing and development or if you are planning to run Loki as a meta-monitoring system.
+2. **Multiple Replicas**: Run Loki with multiple replicas. This mode is useful for high availability and scalability. This mode is less economical than distributed mode, but it is simpler to operate.
+
+Once you have selected choose the appropriate `values.yaml` configuration file below and then continue with the deployment steps.
+  
+### Single Replica
+
+The default Helm chart deploys the following components:
+- Loki (1 replica)
+- Loki Canary (1 DaemonSet)
+- Loki Gateway (1 NGINX replica)
+- Loki Chunk and Result Cache (1 DaemonSet)
+- Minio (optional, if `minio.enabled=true`)
+
+Create the configuration file `values.yaml`:
 
 {{< admonition type="note" >}}
 You must specify `commonConfig.replication_factor: 1` if you are only using 1 replica, otherwise requests will fail.
 {{< /admonition >}}
 
-If you set the `singleBinary.replicas` value to 2 or more, this chart configures Loki to run a *single binary* in a replicated, highly available mode.  When running replicas of a single binary, you must configure object storage.
+```yaml
+loki:
+  commonConfig:
+    replication_factor: 1
+  schemaConfig:
+    configs:
+      - from: 2024-04-01
+        store: tsdb
+        object_store: s3
+        schema: v13
+        index:
+          prefix: loki_index_
+          period: 24h
+  pattern_ingester:
+      enabled: true
+  limits_config:
+    allow_structured_metadata: true
+    volume_enabled: true
+    retention_period: 672h # 28 days retention
+  compactor:
+    retention_enabled: true 
+    delete_request_store: s3
+  ruler:
+    enable_api: true
 
-**Before you begin: Software Requirements**
+minio:
+  enabled: true
+      
+deploymentMode: SingleBinary
 
-- Helm 3 or above. See [Installing Helm](https://helm.sh/docs/intro/install/).
-- A running Kubernetes cluster
+singleBinary:
+  replicas: 1
+
+# Zero out replica counts of other deployment modes
+backend:
+  replicas: 0
+read:
+  replicas: 0
+write:
+  replicas: 0
+
+ingester:
+  replicas: 0
+querier:
+  replicas: 0
+queryFrontend:
+  replicas: 0
+queryScheduler:
+  replicas: 0
+distributor:
+  replicas: 0
+compactor:
+  replicas: 0
+indexGateway:
+  replicas: 0
+bloomCompactor:
+  replicas: 0
+bloomGateway:
+  replicas: 0
+```
+
+In this configuration, we are deploying Loki with MinIO as the object storage. We recommend configuring object storage via cloud provider or pointing Loki at a MinIO cluster for production deployments.
+
+### Multi Replica
+
+The default Helm chart deploys the following components:
```

Comment:
This line and line 30 can't BOTH be true....  Not if the values are different for single and multiple replicas.  I suspect the difference is based on the value set for `replication_factor`?